src/server/constants.ts

```diff
@@ -7,5 +8,8 @@ export const JS_PREFIX = `/js`;
 export const DEBUG = !Deno.env.get("DENO_DEPLOYMENT_ID");
 
 export function bundleAssetUrl(path: string) {
+  if (ASSET_PATH_PREFIX !== "") {
+    return `${ASSET_PATH_PREFIX}${INTERNAL_PREFIX}${JS_PREFIX}${path}`;
+  }
   return `${INTERNAL_PREFIX}${JS_PREFIX}/${BUILD_ID}${path}`;
```

Comment:
Nit: Since by default `ASSET_PATH_PREFIX` is an empty string, so we can just do:

```suggestion
  return `${ASSET_PATH_PREFIX}${INTERNAL_PREFIX}${JS_PREFIX}${path}`;
```

It's one less `if`-branch to worry about.

Reply:
I have tried this in the beginning but I found that it is a tricky one! This will remove the build id for anyone not using a CDN which might cause issues with cache busting on new versions. 

Another option is to keep the build id and set the prefix as the cdn url only, i.e `setAssetPathPrefix(${opts.cdnUrl}/)` but that would mean static files won't be prefixed with a build id when using a CDN which is the same issue just in a different place.

We could also go with what i did in the first iteration where I created two variables: `STATIC_ASSET_PATH_PREFIX` and `INTERNAL_ASSET_PATH_PREFIX`. This will allow us to do `setStaticAssetPathPrefix(${opts.cdnUrl}/${BUILD_ID})` and `setInternalAssetPathPrefix(${opts.cdnUrl}/)`. It felt unnecessary but I think that aligns with the current setup. I also find this option makes storing files on the cdn confusing:
```
/static/[build_id]/...
/_frsh/js/[build_id]/...
```

rather than
```
[build_id]/static/...
[build_id]/_frsh/js/...
```

curious to hear your thoughts.

Reply:
Ohh i totally missed that. You're right, ignore my suggestion.